pkgs/by-name/su/surrealist/package.nix

```
@@ -0,0 +1,66 @@
+{ autoPatchelfHook
+, cairo
+, dpkg
+, fetchurl
+, gdk-pixbuf
+, gobject-introspection
+, lib
+, libsoup
+, makeWrapper
+, pango
+, stdenv
+, webkitgtk
+}:
+
+stdenv.mkDerivation rec {
+  pname = "surrealist";
+  version = "1.11.0";
+
+  src = fetchurl {
+    url = "https://github.com/StarlaneStudios/Surrealist/releases/download/v${version}/surrealist-v${version}-linux.deb";
+    hash = "sha256-ZTm/Qkl96H1IBNvjvCS9XFqYNIK0+KzP6uwbIOwbHgk=";
+  };
+
+  sourceRoot = ".";
+
+  unpackCmd = ''
+    ${dpkg}/bin/dpkg -x $src ./
+  '';
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    dpkg
+    makeWrapper
+  ];
+
+  buildInputs = [
+    cairo
+    gdk-pixbuf
+    gobject-introspection
+    libsoup
+    pango
+    webkitgtk
+  ];
+
+  postInstall = ''
+    install -Dm755 usr/bin/surrealist -t $out/bin
+    install -Dm644 usr/share/applications/surrealist.desktop -t $out/share/applications
+    install -Dm644 usr/share/icons/hicolor/32x32/apps/surrealist.png -t $out/share/icons/hicolor/32x32/apps
+    install -Dm644 usr/share/icons/hicolor/128x128/apps/surrealist.png -t $out/share/icons/hicolor/128x128/apps
+    install -Dm644 usr/share/icons/hicolor/256x256@2/apps/surrealist.png -t $out/share/icons/hicolor/256x256@2/apps
```

nit: no need for write permissions, since nix store is read only.

```suggestion
    install -Dm555 usr/bin/surrealist -t $out/bin
    cp -r usr/share $out
    chmod -R 444 $out/share
```